markdown/inlinepatterns.py

```diff
@@ -151,7 +151,7 @@ def build_inlinepatterns(md_instance, **kwargs):
 AUTOMAIL_RE = r'<([^> \!]*@[^> ]*)>'
 
 # <...>
-HTML_RE = r'(\<([a-zA-Z/][^\>]*?|\!--.*?--)\>)'
+HTML_RE = r'(\<([a-zA-Z/][^\>]*?|\!--.*?--)\>)(?!\>)'
```

Comment:
No, of course not. But the parser is not concerned with whether raw HTML is "valid". Without smarty's angle quotes enabled, I expect the same default behavuor we have now: `<foo>>` becomes `<foo>&gt;`.